### ISSUE TITLE
Compilation Error

### ISSUE THREAD

**Issue by ano45** [Siilwyn/css-declaration-sorter#215]
Hi @Siilwyn,

after upgrading to the latest version I realized that there is a compilation error.

```
Failed to compile.
static/css/main.b90420c1.css from Css Minimizer plugin
TypeError: Cannot read property 'insertAfter' of undefined
    at Array.forEach (<anonymous>)
```

It occurs in the following code block:

```// Add comments back to the nodes they are paired with
  comments.forEach(node => {
    const pairedNode = node.pairedNode;
    node.comment.remove();
    pairedNode.parent['insert' + node.insertPosition](pairedNode, node.comment);
  });
```

It happens in case the pairedNode has no parent.
Then `pairedNode.parent['insert' + node.insertPosition](pairedNode, node.comment);` fails.

**Comment by Siilwyn** [comment]
:wave: could you add a test that fails in a PR?

Or atleast a minimal CSS example that causes this error?

**Comment by ano45** [comment]
I was able to reproduce it with

```
#some-id {
  /*!
 *  comment 1
 */
  div { color: #fff };

  /*!
 * comment 2
 */
  div { color: #fff };

  /*!
 * comment 3
 */
  div { color: #fff };
}
```

but only in a CRA with SASS. Here you can find an example: 
https://github.com/ano45/css-declaration-sorter-error

I was using
```
npm 6.14.11
Node.js v14.15.5.
Ubuntu 20.04.3 LTS
```

**Comment by ayohdev** [comment]
I'm able to replicate this issue as well, using @ano45  example.

`yarn 1.22.15
npm 6.14.13
node 14.17.3

on Gatsby Cloud Build

 ERROR #98123  WEBPACK

Generating JavaScript bundles failed

styles.4fff7283cc0703e03a49.css from Css Minimizer
TypeError: Cannot read property 'insertAfter' of undefined
    at /Users/andrewhewitt/git-templates/gatsby-wp/node_modules/css-declaration-sorter/src/main.js:105:22
    at Array.forEach (<anonymous>)
    at processCss
(/Users/andrewhewitt/git-templates/gatsby-wp/node_modules/css-declaration-sorter/src/main.js:102:12)
    at /Users/andrewhewitt/git-templates/gatsby-wp/node_modules/css-declaration-sorter/src/main.js:37:21
    at async LazyResult.runAsync
(/Users/andrewhewitt/git-templates/gatsby-wp/node_modules/postcss/lib/lazy-result.js:388:13)
    at async cssnanoMinify (eval at transform
(/Users/andrewhewitt/git-templates/gatsby-wp/node_modules/css-minimizer-webpack-plugin/dist/minify.js:34:28),
<anonymous>:51:18)
    at async minify
(/Users/andrewhewitt/git-templates/gatsby-wp/node_modules/css-minimizer-webpack-plugin/dist/minify.js:15:26)
    at async Object.transform
(/Users/andrewhewitt/git-templates/gatsby-wp/node_modules/css-minimizer-webpack-plugin/dist/minify.js:35:18)

File: styles.4fff7283cc0703e03a49.css

`

**Comment by Siilwyn** [comment]
For posterity this is odd behaviour of the `sass-loader` which `css-minimizer-webpack-plugin` is using in the CRA and other examples posted above. SCSS parsing is pretty weird especially when nesting nodes with special comments that it keeps even in compressed mode.

Fixed in version `6.1.4`

**Comment by ano45** [comment]
You made my day! Thanks a lot for the quick fix!
